README.md

```diff
@@ -63,3 +63,22 @@ For example:
 Ozone FOSS requires you to log into each component separately.
 
 💡 **Did you know?** Ozone Pro comes with single sign-on (SSO) and all its integration layer is secured with OAuth2.
+
+## Create Your Own Distribution
```

Comment:
Would you be able to add this section as a PR in the https://github.com/ozone-his/ozone-docs repo instead?